computeResource.go

```diff
@@ -152,6 +152,19 @@ type computerResourceResponse struct {
 	Data ComputeResource `json:"data"`
 }
 
+func (s *ComputeResourcesService) List(ctx context.Context) ([]ComputeResource, error) {
```

Comment:
I mean that we should return paginated response and let users iterate other it.

Reply:
Got you.